docs/multi-agent-manager.md

```diff
@@ -79,13 +71,13 @@ component_overrides = [{
       "not_after": '2022-01-01',
```

Comment:
These are the defaults now, so let's just remove the overrides? Although there is merit in demonstrating the idea.